qa/suites/rgw/dbstore/tasks/rgw_s3tests.yaml

```diff
@@ -0,0 +1,23 @@
+tasks:
+- install:
+- ceph:
+- rgw: [client.0]
+- s3tests:
+    client.0:
+      dbstore_tests: True
+      force-repo: https://github.com/soumyakoduri/s3-tests.git
```

Comment:
This will be set  back to default once https://github.com/ceph/s3-tests/pull/441 gets merged.